client/lib/upgrades/notices.jsx

```diff
@@ -0,0 +1,52 @@
+/**
+ * External dependencies
+ */
+import React from 'react'; // eslint-disable-line no-unused-vars
+import flatten from 'lodash/flatten';
+import values from 'lodash/values';
+
+/**
+ * Internal dependencies
+ */
+import i18n from 'lib/mixins/i18n';
+import notices from 'notices'
+import ValidationErrorList from 'notices/validation-error-list';
+
+function getErrorFromApi( errorMessage ) {
```

Comment:
Thanks for moving this. I'm glad my name won't be associated with this anymore ;)